src/main/webapp/WEB-INF/context/applicationContext.xml

```diff
@@ -71,12 +71,17 @@
     </util:map>
     
     <!-- NEWS ADAPTER REGISTRATIONS -->
-    
-    <bean id="romeProcessor" 
+
+    <bean id="romeProcessor"
         class="org.jasig.portlet.newsreader.processor.RomeNewsProcessorImpl"
-        p:imageTypes-ref="imageTypes" p:videoTypes-ref="videoTypes" 
-        p:policies-ref="antisamyPolicies"/>
-    
+        p:imageTypes-ref="imageTypes" p:videoTypes-ref="videoTypes"
+        p:policies-ref="antisamyPolicies" p:entriesPerPage="10"/>
```

Comment:
So entriesPerPage is something you set in the app context at build time?  Looks like something you should set in portlet preferences through the admin UI.